extensions/vi-mode/tests/utils.lisp

```diff
@@ -0,0 +1,298 @@
+(defpackage :lem-vi-mode/tests/utils
+  (:use :cl
+        :lem)
+  (:shadow :with-current-buffer)
+  (:import-from :rove
+                :form-description
+                :diag
+                :testing)
+  (:import-from :lem-base
+                :*current-buffer*)
+  (:import-from :lem-fake-interface
+                :with-fake-interface)
+  (:import-from :lem-vi-mode
+                :vi-mode)
+  (:import-from :lem-vi-mode/core
+                :*current-state*
+                :state-keymap
+                :normal
+                :insert
+                :current-state
+                :ensure-state)
+  (:import-from :lem-vi-mode/visual
+                :visual-line
+                :visual-block
+                :apply-visual-range)
+  (:import-from :cl-ppcre
+                :scan
+                :regex-replace)
+  (:import-from :alexandria
+                :remove-from-plistf
+                :with-gensyms
+                :once-only
+                :appendf
+                :if-let)
```

Comment:
It seems that there are symbols that have been imported but not used.
Can you check with `M-x lisp-organize-import`?

Reply:
Why haven't I known this feature?
This must be one of the good reasons to recommend people to use Lem.